### PR TITLE
Add publications tab and fix old links

### PR DESCRIPTION
**Pull request by chrzyki**
This addresses #6. The list is not done yet, but I've opened the PR for discussion:

- what else should be in general papers?
- is the general structure okay? Should there be more/less info?

Happy to add more things & modify.

### CODE REVIEW

File: publications.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="chrome=1">
+    <title>CLDF - Cross-Linguistic Data Formats - Examples</title>
```
Comment (xrotwang) [inline publications.html]: Wrong title, I guess.

File: publications.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    <meta name="viewport" content="width=device-width">
+</head>
+
+<body class="site">
+<header class="site-header">
+    <nav>
+        <div class="sticky-nav">
+            <div class="container-nav">
+                <div class="nav-container">
+                    <div class="navbar-brand"><a href="http://cldf.clld.org">
```
Comment (xrotwang) [inline publications.html]: Should be switched to `https://...`

File: publications.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+                </div>
+                <div class="column-50 gray-box">
+                    <h2>Catalogs</h2>
+                    <h3>Glottolog</h3>
+                    <ul>
+                        <li>Paper</li>
+                        <li>Data:
+                            <a href="https://doi.org/10.5281/zenodo.3554959"><img
+                                    src="https://zenodo.org/badge/DOI/10.5281/zenodo.3554959.svg"
+                                    alt="DOI"></a>
```
Comment (xrotwang) [inline publications.html:74]: I'd add short descriptions of what to expect/find, e.g. "Glottolog data releases on Zenodo", or "pyglottolog Python library"

File: publications.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                </div>
+            </div>
+            <div class="service-flexrow pad-top">
+                <div class="column-50 gray-box">
+                    <h2>Datasets</h2>
+                    <ul>
+                        <li><a href="https://zenodo.org/communities/cldf-datasets/">Zenodo community
+                            for various kinds of datasets in CLDF format</a></li>
+                        <li><a href="https://zenodo.org/communities/lexibank/">Zenodo community for
+                            Lexibank datasets</a></li>
```
Comment (xrotwang) [inline publications.html:124]: Add dictionaria community.

**Review by xrotwang — CHANGES_REQUESTED** [review]

**Comment by chrzyki** [comment]
Thanks, updated!

File: publications.html [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+            <h2 class="headline text-center">CLDF-related Publications</h2>
+        </div>
+    </section>
+    <section class="section-main">
+        <div class="container pad-top">
+            <div class="service-flexrow">
+                <div class="column-50 gray-box">
+                    <h2>General papers</h2>
+                    <ul>
+                        <li><a href="https://digling.org/evobib/?bibtex=Forkel2018a">Cross-Linguistic
```
Comment (xrotwang) [inline publications.html]: I think I'd link the DOI - maybe evobib in addition. But the main idea here is getting to the paper, not necessarily the bibtex formatted reference, I think.

**Review by xrotwang — CHANGES_REQUESTED** [review]

File: publications.html (hunk 5 above)
Comment (chrzyki) [reply publications.html]: Updated, thanks.

**Pull request merged**